deployment/packagebuild/packages.d/vnet/openvnet-vnmgr/post-install

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+user="vnet-vnmgr"
+logfile="/var/log/openvnet/vnmgr.log"
+
+if ! id "$user" > /dev/null 2>&1 ; then
+    adduser --system --no-create-home --shell /bin/false "$user"
+fi
+
+touch "$logfile"
```

Comment:
+1

If the openvnet-common RPM creates the folder, then the upstart jobs don't need to check for it. It's kind of unrelated to the problem at hand though. Can either be fixed here or in a next pull request.